Source/JavaScriptCore/offlineasm/x86.rb

```diff
@@ -177,66 +177,43 @@ def supports8BitOnX86
     end
 
     def x86GPR
-        if isX64
```

Comment:
Hmm, seems like `isX64` should be removed altogether if it just throws in the false case now...?

Reply:
Sounds good.

Reply:
Maybe the file should be renamed from x86.rb to x86_64.rb (or x64.rb) to make it clear that it does not support 32-bit x86 anymore.  I suggest doing that in a subsequent patch by itself.

Reply:
Sounds good too!